docs/deployments/free5gc-with-bgp/Makefile

```diff
@@ -16,6 +16,7 @@ free5gc:
 calico:
 	kubectl apply -f $(HELM_VALUES_DIR)/../manifests/calico-pools.yaml
 	kubectl apply -f $(HELM_VALUES_DIR)/../manifests/calico-bgp.yaml
+	kubectl patch felixconfiguration default --type='merge' -p '{"spec":{"removeExternalRoutes":false}}'
```

Comment:
Let's add a comment why this patch is needed

Reply:
done